knotx-core/src/main/java/com/cognifide/knotx/placeholder/UriHelper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Knot.x - Reactive microservice assembler
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+package com.cognifide.knotx.placeholder;
+
+import org.jsoup.nodes.Element;
+import org.springframework.util.PropertyPlaceholderHelper;
+
+import io.vertx.core.http.HttpServerRequest;
+
+public final class UriHelper {
```

Comment:
Please implement this class as singleton bean (not helper class), then you can write proper unit tests for this class.

http://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html